src/error.js

```diff
@@ -630,6 +637,15 @@ export function getErrorReportData(
   data['ae'] = accumulatedErrorMessages.join(',');
   data['fr'] = self.location.originalHash || self.location.hash;
 
+  // TODO(https://github.com/ampproject/error-tracker/issues/129): Remove once
+  // all clients are serving a version with pre-throttling.
+  if (data['rt'] === 'production') {
```

Comment:
I don't think we can ever remove this, since we don't wan the reporting function to throttle canary.

Reply:
As in, we can remove the "prethrottle" param and just use `data['rt']` directly in the conditional above. We only need to explicitly provide "prethrottle" while some versions aren't doing it.

Reply:
Ahh